addons/web/static/src/core/hotkeys/hotkey_service.js

```diff
@@ -19,11 +19,12 @@ const NAV_KEYS = [
     "enter",
     "escape",
 ];
-const MODIFIERS = new Set(["control", "shift"]);
+const MODIFIERS = new Set(["alt", "control", "shift"]);
 const AUTHORIZED_KEYS = new Set([...ALPHANUM_KEYS, ...NAV_KEYS]);
 
 export const hotkeyService = {
     dependencies: ["ui"],
+    overlayModifier: "alt",
```

Comment:
comment: "Be aware that all odoo hotkeys are designed with this modifier in mind, so changing the overlay modifier may conflict with some shortcuts.